mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/rg-template.json

```diff
@@ -0,0 +1,346 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "projectPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Prefix used for naming resources."
+            },
+            "defaultValue": "mlos"
+        },
+        "vmSKU": {
+            "type": "string",
+            "defaultValue": "Standard_D2s_v3",
+            "metadata": {
+                "description": "VM SKU"
+            }
+        },
+        "vmAdminUsername": {
+            "type": "string"
+        },
+        "sshPublicKeys": {
+            "type": "array"
+        },
+        "storageAccountLocations": {
+            "type": "array",
+            "defaultValue": [
+                "[resourceGroup().location]"
+            ]
+        },
+        "fileShareName": {
+            "type": "string",
+            "defaultValue": "mlos-file-share"
+        },
+        "vmLocation": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]"
```

Comment:
Will this create a VM in each location that we need storage as well?

Reply:
Probably only need the scheduler VM in a single location, but we will need storage in each location that we want to run experiments.

Reply:
The VM will not follow the storage locations, only a single VM in this location as specified.